### DISCUSSION TITLE
Ahbap şirketler

### DISCUSSION THREAD

**Discussion by webuti** [acikkaynak/deprem-yardim-projesi#32] [Ideas]
Şuan faydali olan ve daha sonrada faydali olabilecek şirketleri sektör bazlı olarak sirket rehberi yapalim fark olarak bu şirketler hem daha once yaptıkları işbirlikleri sistemde yer alsın, ve sonra afet durumlarında STK'lara vereceğimiz panelden istek iletilecek örnek veriyorum çadır ihtiyacı 1000 adet, şirketlere bildirim mail sms hepsi gidecek hangisi ulaşırsa onlar sistemden ne kadarını ne fiyata(yada bağış) karşılayabilecek lerini yazıp gönderecek STK onaylayıp göndereceği adresi seçecek, STK'lar kimden nereye ne kadar neyden gittiğini görmüş olacak lojistik sürecinde katkı saglanacak.

**Comment by AhmetMTambay** [comment]
Yardımların STK deposunda fiziksel olarak toplamadan hedef bölge yakınında toplandıktan sonra tasnif edilerek veya hedef bölgede ayrılarak dağıtılması bölgeye giden toplam yardımların takibini sağlayabilir. Bölgede tasnifleme ve dağıtım için yeterli kişi bulunmaması riski olabilir.

**Comment by webuti** [comment]
gönderileceği adres bölümü STK`ya ait bir bölge olabilir, nereye neyi gönderdiklerini ve içeriğinin ne olduğunu takip edebilmeyi takip etmeyi kolaylaştırmak icin bahsettim. Onay durumu yada süreç durumu da eklenebilir

**Comment by canotur** [comment]
Bu konuya ben de katılıyorum. Bu konuda tüm uygulamalar birbiriyle entegre çalışabilir diye düşünüyorum. Eğer ki ortak bir altyapı oluşturulur ve herkesin burayı beslemesi sağlanacak şekilde tüm ekipler bir araya gelip bir çözüm üretilirse neden yapılamasın. Hatta bilginin nereden geldiği de önemli olmaz bu durumda. Cep Telefonu numarası üzerinden hesap yönetimi yapılarak kullanılabilir. Fakat bu konuda hem e-ticaret firmalarını hem de STK'ları da işin içine katarak ortak bir çalışmaya ikna etmek ve birlikte çalışmak. Konuyla ilgili yazımı paylaşıyorum.

Ama konu başlığını Ahbap olarak değil de Ortak Market ve Lojistik Uygulaması gibi bir şey yapabiliriz.

Hatta sonrasında haftalar sonra hala eğer gerçekten tahmin edildiği gibi yardıma devam etmemiz gerekirse, e-ticaret firmaları askıda fatura gibi ücretli olarak alınan siparişlerin ödemelerini hayırseverler tarafından yapılmasını sağlayabilir.

https://www.linkedin.com/posts/canotur_afad-akut-ahbap-activity-7029847651210674176-KY-B?utm_source=share&utm_medium=member_desktop

https://www.linkedin.com/feed/update/urn:li:activity:7029847651210674176?commentUrn=urn%3Ali%3Acomment%3A%28activity%3A7029847651210674176%2C7029940293252571136%29

**Comment by webuti** [comment]
Benim aslinda aklimdaki kategoriler ilk etapta şunlardı, Vinç şirketleri, Kepçe şirketleri,, çadır,solar, drone, konteyner,, termal kamera, pil, jeneratör ayni sirket birden fazla hizmet verebildiği kategoriyi de seçebilecek .Ahbap ismini isbirlikcisirkerler algisi icin yazdim baska isimlerde bulunabilir.

**Comment by ramazansancar** [comment]
Fikriniz için teşekkürler. Legal ekibi ile birlikte değerlendirmemiz sonrasında şu notları aktarabilirim.

"Legale tabi değil ancak bu çalışma halihazırda ihtiyaç listesi vb projeler altında ilerliyor"

Dilerseniz projeyi geliştirme noktasında ilgili projelere pr veya issues açabilirsiniz.
Projeler hakkında bilgiye buradan ulaşabilirsiniz: https://afet.org/
